src/ml/neural_net/image_augmentation.hpp

```diff
@@ -269,7 +269,57 @@ class resize_only_image_augmenter final: public image_augmenter {
   options opts_;
 };
 
+/**
+ * An abstract class that inherits from image_augmenter used to convert
+ * input images, annotations and predictions to shared_float_arrays for
+ * tf_image_augmenter.
+ * Subclass must be written for it if needed. The subclass must implement the
```

Comment:
Not sure, but you may need to leave a blank line (with just a `*`) so that Doxygen won't interpret this as one big paragraph. Assuming you intend this to be two paragraphs.